config/environments/production.rb

```diff
@@ -41,7 +41,7 @@
   config.log_level = ENV.fetch("RAILS_LOG_LEVEL", "info")
 
   # Prevent health checks from clogging up the logs.
-  config.silence_healthcheck_path = "/up"
+  config.silence_healthcheck_path = /^\/status/
```

Comment:
Could you drop a comment about that, so we don't forget?

Reply:
The reason I didn't do that is because it will shortly be a crufty comment that we're likely to forget about removing. However, if you think it should be added, I'm happy to.

Reply:
Defer to your judgment on this. I suppose this PR will serve as documentation.